mobile-app/lib/features/components/recent_address_list_item.dart

```diff
@@ -76,7 +79,7 @@ class _RecentAddressListItemState extends State<RecentAddressListItem> {
             widget.address,
             style: TextStyle(
               color: Colors.white.useOpacity(0.60),
-              fontSize: 11,
+              fontSize: isTablet ? 16 : 11,
```

Comment:
this would be a lot better with a global call - get me the font size 11 for example, and the global function would check isTablet and return 16 or 11 depending. Something like that. 

It's not good to pass this around all over the place. 

Could also check what the common ways are that this is solved. 

We should not have if statements all over the codebase. If it's in one or two locations, that's ok, else it should be in dedicated classes that abstract these things out.